core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationRepositorySpec.scala

```diff
@@ -86,4 +86,59 @@ class GoogleProjectRegistrationRepositorySpec extends AnyFlatSpec with TestDrive
 
   }
 
+  it should "retrieve a google project registration by id" in withDefaultTestDatabase {
```

Comment:
The functions added to `GoogleProjectRegistrationRepository` are only pass throughs. The test cases added here should be covered by `GoogleProjectRegistrationComponent` or vice versa. I don't think you need the same coverage in both places. Make sure you have the right coverage in one place or the other but I don't think you need both.